tests/lib/eslint.js

```diff
@@ -2193,6 +2193,11 @@ describe("eslint", function() {
             assert.equal(messages.length, 0);
         });
 
+        it("should send sourceType \"module\" to escope when there is an ecmaFeatures.modules (plural) and ecmaFeatures.blockBindings attributes in the config", function() {
```

Comment:
unless we are ok with one failing test, because with only the modules attribute the ecmaVersion is not changed to 6